app/models/evaluation.rb

```diff
@@ -1,24 +1,11 @@
 class Evaluation < ApplicationRecord
-  self.table_name = 'ertekelesek'
-  self.primary_key = :id
-
   before_save :set_default_values
 
-  alias_attribute :entry_request_status, :belepoigeny_statusz
-  alias_attribute :timestamp, :feladas
-  alias_attribute :point_request_status, :pontigeny_statusz
-  alias_attribute :date, :semester # This alias is misleading
-  alias_attribute :justification, :szoveges_ertekeles
-  alias_attribute :last_evaulation, :utolso_elbiralas
-  alias_attribute :last_modification, :utolso_modositas
-  alias_attribute :reviewer_user_id, :elbiralo_usr_id
-  alias_attribute :group_id, :grp_id
-  alias_attribute :creator_user_id, :felado_usr_id
-  alias_attribute :principle, :pontozasi_elvek
+  alias_attribute :date, :semester
 
-  belongs_to :group, foreign_key: :grp_id
-  has_many :point_requests, foreign_key: :ertekeles_id, inverse_of: :evaluation
-  has_many :entry_requests, foreign_key: :ertekeles_id
+  belongs_to :group
+  has_many :point_requests
```

Comment:
Don't we need the `inverse_of` anymore?

Reply:
You are right with the issue, but we are using Trello currently, so I create there one, until we agree moving it to Github.

Reply:
I am really sure that we are 'using' anything. I mean because the developing process isn't too active.
And I think that would be a better choice if we are starting the task management from the beginning.
But its really out of this PR scope :D

Reply:
Yeah, I agree, but let this decision for the young ones and also we have a lot of things in Trello, so I'd move them here before starting using issues here.